docs/src/main/sphinx/release/release-332.md

```diff
@@ -60,7 +60,7 @@
   `hive.metastore.glue.endpoint-url` configuration property. ({issue}`3239`)
 - Add experimental file system caching. This can be enabled with the `hive.cache.enabled` configuration property. ({issue}`2679`)
 - Support reading files compressed with newer versions of LZO. ({issue}`3209`)
-- Add support for {ref}`alluxio-catalog-service`. ({issue}`2116`)
+- Add support for Alluxio Catalog Service. ({issue}`2116`)
```

Comment:
Unrelated .. probably need to rebase

Reply:
Still fails after rebase: Warning, treated as error:
/docs/src/main/sphinx/release/release-332.md:63:undefined label: alluxio-catalog-service